### PR TITLE
[WIP]Nfs Driver code for fileshare support

### PR DESCRIPTION
**Pull request by PravinRanjan10**
**What this PR does / why we need it**:
NFS driver to support fileshare creation and mounting.
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: contrib/drivers/filesharedrivers/nfs/cli.go [hunk 1]
```diff
@@ -0,0 +1,234 @@
+// Copyright (c) 2018 Huawei Technologies Co., Ltd. All Rights Reserved.
```
Comment (leonwanghui) [inline contrib/drivers/filesharedrivers/nfs/cli.go]: Please update the copyright.

File: contrib/drivers/filesharedrivers/nfs/nfs.go [hunk 2]
```diff
@@ -1,50 +1,209 @@
 // Copyright 2019 The OpenSDS Authors.
+// Copyright (c) 2017 Huawei Technologies Co., Ltd. All Rights Reserved.
```
Comment (leonwanghui) [inline contrib/drivers/filesharedrivers/nfs/nfs.go]: Copyright

File: pkg/model/fileshare.go [hunk 3]
```diff
@@ -84,7 +84,7 @@ type FileShareSpec struct {
 	SnapshotId string `json:"snapshotId,omitempty"`
 
 	// ExportLocations of the fileshare.
-	ExportLocations []string `json:"exportLocations,omitempty"`
+	ExportLocations string `json:"exportLocations,omitempty"`
```
Comment (leonwanghui) [inline pkg/model/fileshare.go]: I don't understand why this is changed to `string`

Comment (PravinRanjan10) [reply pkg/model/fileshare.go]: Currently we are testing on only one network, so made it string. I will remove string and make array. Thanks

**Comment by PravinRanjan10** [comment]
Closing this PR. I will open a new with latest codes

**Pull request closed without merging**